docs/user/openaev-registration.md

```diff
@@ -0,0 +1,15 @@
+# OpenAEV Platform Registration
+
+
+To utilize features such as one-click deployments, your OpenAEV platform must be registered (available from OpenAEV 2.0.0).
+
+The registration process is initiated from your OpenAEV platform. For more detailed information, please refer to the [OpenAEV documentation](https://docs.openbas.io/latest/administration/hub).
```

Comment:
Documentation link needs to be changed after OpenAEV renaming

Reply:
So we merge this documentation afterwards or you create a follow-up issue ?